content/en/api_catalog/endpoint_discovery.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Endpoint Discovery from APM
+kind: documentation
+is_beta: true
+further_reading:
+- link: "/tracing/api_catalog/"
+  tag: "Documentation"
+  text: "Datadog API Catalog"
+aliases:
+    - /tracing/api_catalog/endpoint_discovery/
+---
+
+## Overview
+If a service has a supported tracer installed, the API Catalog will automatically be filled by all endpoints discovered from this service. 
+
+In order to check compatibility, you can visit the ‘troubleshoot’ page on the app by pressing the ‘learn more’ button
+{{< img src="tracing/api_catalog/api-catalog-discovery-learn-more.png" alt="ALT TEXT" style="width:30%;text-align: left;" >}}
+
+## Providing endpoint path manually
+Add the `datadog.api_catalog.route` tag to force endpoint discovery by the API Catalog.
+This tag should contain the matched route, that is, the path template in the format used by the respective server framework.
+
+Note: API Catalog will still filter out spans that do not contain the following tags:
+* `http.method`
+* `http.status_code`
```

Comment:
```suggestion
<div class="alert alert-info">API Catalog filters out spans that do not contain the following tags:<ul>
<li> <code>http.method</code>
<li> <code>http.status_code</code></ul>
</div>
```
This is how we can make alert boxes in the docs that look like this:
![image](https://github.com/DataDog/documentation/assets/84536271/68d70773-c1cc-4ffa-9e3b-fafd1c562727)